R/browse.R

```diff
@@ -59,6 +59,14 @@ browse_travis <- function(package = NULL, ext = c("org", "com")) {
   view_url(sub("github.com", travis_url, gh))
 }
 
+#' @export
```

Comment:
Can you please add a bullet point to the documentation above, following the existing pattern?